features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/BreadcrumbTooBigScenario.kt

```diff
@@ -0,0 +1,23 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+
+class BreadcrumbTooBigScenario(
+    config: Configuration,
+    context: Context,
+    eventMetadata: String
+) : Scenario(config, context, eventMetadata) {
+
+    init {
+        config.maxStringValueLength = 2000000
+    }
+
+    override fun startScenario() {
+        super.startScenario()
+
+        Bugsnag.leaveBreadcrumb("test" + "*".repeat(1100000))
+        Bugsnag.notify(generateException())
```

Comment:
It would be good if at least one of these scenarios failed with an unhandled error, just to ensure that path functions as expected.